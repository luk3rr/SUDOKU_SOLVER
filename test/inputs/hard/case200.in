200007504 040000007 000020680 008010000 000200006 530090000 050000200 004070001 800400900 
