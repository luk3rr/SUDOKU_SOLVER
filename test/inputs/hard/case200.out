Solving the following grid:
2 0 0 | 0 0 7 | 5 0 4 
0 4 0 | 0 0 0 | 0 0 7 
0 0 0 | 0 2 0 | 6 8 0 
------+-------+------
0 0 8 | 0 1 0 | 0 0 0 
0 0 0 | 2 0 0 | 0 0 6 
5 3 0 | 0 9 0 | 0 0 0 
------+-------+------
0 5 0 | 0 0 0 | 2 0 0 
0 0 4 | 0 7 0 | 0 0 1 
8 0 0 | 4 0 0 | 9 0 0 


Solution found :')

2 8 9 | 1 6 7 | 5 3 4 
6 4 3 | 8 5 9 | 1 2 7 
1 7 5 | 3 2 4 | 6 8 9 
------+-------+------
4 6 8 | 5 1 3 | 7 9 2 
7 9 1 | 2 4 8 | 3 5 6 
5 3 2 | 7 9 6 | 4 1 8 
------+-------+------
9 5 7 | 6 8 1 | 2 4 3 
3 2 4 | 9 7 5 | 8 6 1 
8 1 6 | 4 3 2 | 9 7 5 

Algorithm: BFS
Total time: 750 ms
Total expanded states: 121488
